DataFormats/TrackerCommon/interface/PixelPannelType.h

```diff
@@ -1,7 +1,7 @@
 #ifndef DataFormats_SiPixelDetId_PixelPannelType_H
```

Comment:
Just noticing that in the other moved headers you also updated the header guard....